es6-shim.js

```diff
@@ -1269,7 +1269,13 @@
       if (value === 0) { return value; }
       var negate = value < 0, result;
       if (negate) { value = -value; }
-      result = Math.pow(value, 1 / 3);
+      if (value === Infinity) {
```

Comment:
-Infinity is converted to Infinity 2 lines above, so the "true" branch will be involved, and the "sign" will be set before return